src/client/components/CVreviewsPage/CVreviews.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useStorage } from '../../hooks/fileUploader';
+
+export function CVreviews({ uploadedFile }) {
+  const { url } = useStorage(uploadedFile);
+
+  return (
+    <iframe
+      title={url.name}
+      style={{ width: '50%', height: '700px' }}
+      scrolling="yes"
+      src={`${url} #toolbar=0`}
```

Comment:
```suggestion
      src={`https://docs.google.com/viewer?url=${url}&embedded=true`}
```

Reply:
this is not working. It shows me "No Preview Available" 
<img width="1440" alt="Screenshot 2021-02-11 at 22 42 36" src="https://user-images.githubusercontent.com/54910388/107702418-7b1ac180-6cba-11eb-8435-9dbf3d9ee06c.png">